src/main/java/com/vinylteam/vinyl/dao/jdbc/JdbcUserDao.java

```diff
@@ -1,6 +1,89 @@
 package com.vinylteam.vinyl.dao.jdbc;
 
 import com.vinylteam.vinyl.dao.UserDao;
+import com.vinylteam.vinyl.dao.jdbc.mapper.UserRowMapper;
+import com.vinylteam.vinyl.entity.User;
+import org.postgresql.ds.PGSimpleDataSource;
+import org.postgresql.util.PSQLException;
+
+import java.sql.*;
 
 public class JdbcUserDao implements UserDao {
+
+    private final PGSimpleDataSource dataSource;
+    private final String COUNT_ALL = "SELECT COUNT(*) FROM public.users";
+    private final String FIND_BY_EMAIL = "SELECT * FROM public.users" +
+            " WHERE email=?";
+    private final String INSERT = "INSERT INTO public.users" +
+            " (\"email\", \"password\", \"salt\", \"iterations\", \"role\")" +
+            " VALUES (?, ?, ?, ?, ?)";
```

Comment:
why we use "" in name?

Reply:
You mean names of columns? Just a precaution - to not forget "" when I work with columns' names with spaces.

Reply:
I think that the use of double quotes around the column names is unnecessary. Please delete them.